lib/manager/components/reporter/components/RouteLayout.js

```diff
@@ -98,18 +101,33 @@ export default class RouteLayout extends Component<Props> {
     // order to see items 1 - 4).
     const extraPage = +(routeOffset % PAGE_SIZE > 0)
     const activePage = Math.floor(routeOffset / PAGE_SIZE) + 1 + extraPage
+    const numPages = Math.ceil(numRoutes / PAGE_SIZE)
+
+    const paginationItems = []
+    for (let pageNum = 1; pageNum <= numPages; pageNum++) {
+      paginationItems.push(
+        <Pagination.Item
+          active={pageNum === activePage}
+          key={pageNum}
+          onClick={(e) => this._onPaginationSelect(+e.target.text)}
```

Comment:
Convert the handle to a class method.

Reply:
Done in https://github.com/ibi-group/datatools-ui/pull/995/commits/d74ef3ffa5e85f14507bea7c1032744e0a74753b